compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -128,6 +128,48 @@ if (cg->profiledPointersRequireRelocation() && secondChild->getOpCodeValue() ==
    }
 #endif
 
+   if (secondChild->getOpCode().isLoadConst())
+      {
+      int64_t secondChildValue = is64bit ? secondChild->getLongInt() : secondChild->getInt();
+      if ((cc == TR::CC_EQ || cc == TR::CC_NE)
+            && secondChildValue == 0)
+         {
+         TR::InstOpCode::Mnemonic op;
+         bool isEqualCmp = TR::ILOpCode::isEqualCmp(node->getOpCode().getOpCodeValue());
+         bool isNotEqualCmp = TR::ILOpCode::isNotEqualCmp(node->getOpCode().getOpCodeValue());
+         TR_ASSERT_FATAL(isEqualCmp || isNotEqualCmp, "The node has to be isEqualCmp or isNotEqualCmp");
+         if (isEqualCmp)
+            op = is64bit ? TR::InstOpCode::cbzx : TR::InstOpCode::cbzw;
+         else
+            op = is64bit ? TR::InstOpCode::cbnzx : TR::InstOpCode::cbnzw;
+
+         TR::LabelSymbol *dstLabel = node->getBranchDestination()->getNode()->getLabel();
```

Comment:
Variables `dstLabel` and `result` are declared twice in this function for different scopes, and that can be confusing.
Please move the declaration of the variables to the beginning of the function.

Reply:
`dstLabel`, `result` and `deps` declaration of the variables are changed to the beginning of the function.